src/Thenewboston.Tests/Bank/Api/ConnectedBanksServiceTests.cs

```diff
@@ -20,15 +20,15 @@ public class GetBanksAsync
             [Fact]
             public async void ListOfBanksIsReturned()
             {
-                var expectedResponseModel = new PaginatedResponseModel<BankResponseModel>
+                var expectedResponseModel = new PaginatedResponseModel<BankResponse>
```

Comment:
Do you think we should maybe name this model "BankNodeResponse" since a bank can make multiple types of responses (Like confirmation blocks, etc)?